tests/integration/test_backups.py

```diff
@@ -115,7 +127,9 @@ async def test_build_and_deploy(ops_test: OpsTest, mysql_charm_series: str) -> N
 
     logger.info("Deploying s3 integrator")
 
-    await ops_test.model.deploy(S3_INTEGRATOR, channel=S3_INTEGRATOR_CHANNEL)
+    await ops_test.model.deploy(
+        S3_INTEGRATOR, channel=S3_INTEGRATOR_CHANNEL, config={"path": path}
+    )
```

Comment:
why was this change made? i believe we run `sync-s3-credentials` before  creating/restoring a backup to correctly set the path